qcut/docs/issues/video_play/consolev4.md

```
@@ -0,0 +1,177 @@
+blob-manager.ts:193 [BlobManager] 📉 Released: blob:app://./2e67ac73-c2ca-42be-9f8d-a52f8a91de8e
+blob-manager.ts:194   📍 Created by: media-store-display
+blob-manager.ts:195   🔄 Released by: VideoPlayer-unmount
+blob-manager.ts:196   📊 Remaining refs: 1
+blob-manager.ts:71 [BlobManager] ♻️ Reusing URL (instance match): ecbb2587-d623-2eb2-109f-e8306a658615
+blob-manager.ts:74   📍 Original source: media-store-display
+blob-manager.ts:75   🔄 Requested by: VideoPlayer
+blob-manager.ts:76   📊 Ref count: 2
+video-player.tsx:184 [VideoPlayer] Using blob URL for ecbb2587-d623-2eb2-109f-e8306a658615: blob:app://./2e67ac73-c2ca-42be-9f8d-a52f8a91de8e
+blob:app://./2e67ac73-c2ca-42be-9f8d-a52f8a91de8e:1  Failed to load resource: net::ERR_UPLOAD_FILE_CHANGED
+index.html:68 🔧 Suppressed blob network error for: VIDEO
+html2canvas.esm.js:5347 Canvas2D: Multiple readback operations using getImageData are faster with the willReadFrequently attribute set to true. See: https://html.spec.whatwg.org/multipage/canvas.html#concept-canvas-will-read-frequently
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)